com.one_ware.OneWare.yml

```diff
@@ -0,0 +1,49 @@
+app-id: com.one_ware.OneWare
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: oneware
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --persist=.local/share/OneWareStudio
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --filesystem=host
```

Comment:
this is a very broad file system permission?

Reply:
I was looking at other examples for IDEs and in my opinion it does make sense to have broad file system permissions for a code editor, similar to Rider or VSCode?

Reply:
I changed it tom home already. This will force users to install their Quartus (or other FPGA toolchains) in home aswell. At least for quartus thats the default install location anyways so its not so bad.
![image](https://github.com/flathub/flathub/assets/25281882/a5168698-1c94-475d-b0ed-7a046925d5d0)